applications: nrf_desktop: Remove modules/Kconfig.ble file (#10188)
nrfconnect/sdk-nrf: applications/nrf_desktop/doc/ble_conn_params.rst

*************

The module requires the basic Bluetooth configuration, as described in :ref:`nrf_desktop_bluetooth_guide`.
Make sure that both :ref:`CONFIG_DESKTOP_ROLE_HID_DONGLE <config_desktop_app_options>` and :ref:`CONFIG_DESKTOP_BT_CENTRAL <config_desktop_app_options>` options are enabled.

> The |ble_conn_params| is enabled by the :ref:`CONFIG_DESKTOP_BLE_CONN_PARAMS_ENABLE <config_desktop_app_options>` option, which is implied by :ref:`CONFIG_DESKTOP_BT_CENTRAL <config_desktop_app_options>` together with other features used by a HID dongle that forwards HID reports received over Bluetooth LE.

Reviewer: ```suggestion
The |ble_conn_params| is enabled by the :ref:`CONFIG_DESKTOP_BLE_CONN_PARAMS_ENABLE <config_desktop_app_options>` option. 
The option is implied by :ref:`CONFIG_DESKTOP_BT_CENTRAL <config_desktop_app_options>` together with other features used by an HID dongle that forwards the HID reports received over Bluetooth LE.
```
